### ISSUE TITLE
Passing directory arguments results in `Could not get the ContentType` errors since 2.8.0

### ISSUE THREAD

**Issue by silverwind** [editorconfig-checker/editorconfig-checker#326]
Since 2.8.0, this module seems to no longer work when passing directories as arguments:

```bash
$ mkdir foo
$ touch foo/bar.txt
$ go run github.com/editorconfig-checker/editorconfig-checker/cmd/editorconfig-checker@2.7.0 foo
$ go run github.com/editorconfig-checker/editorconfig-checker/v2/cmd/editorconfig-checker@2.8.0 foo
Could not get the ContentType of file: bar.txt
stat bar.txt: no such file or directory
```

**Comment by silverwind** [comment]
Another related problem is that [exit code seems to be 0](https://github.com/go-gitea/gitea/actions/runs/7759307758/job/21163096228#step:5:9) with this kind of error, but I would have expected it to be non-zero when one or more errors are present.

**Comment by greut** [comment]
@silverwind it was likely introduced by me there: #292 

#327 seems to fix this, you may try it using the commit as below.

```console
$ go run github.com/editorconfig-checker/editorconfig-checker/v2/cmd/editorconfig-checker@a7a8d55e3aaecd1b5f1d4c98df01bc12ccf857b7
```

**Comment by janedbal** [comment]
Will you tag the fix? Thanks
